app/routes/apply/eligibility/new-claim/same-journey-as-last-claim.js

```diff
@@ -0,0 +1,56 @@
+const UrlPathValidator = require('../../../../services/validators/url-path-validator')
+const referenceIdHelper = require('../../../helpers/reference-id-helper')
+const ValidationError = require('../../../../services/errors/validation-error')
+const SameJourneyAsLastClaim = require('../../../../services/domain/same-journey-as-last-claim')
+const getLastClaimDetails = require('../../../../services/data/get-last-claim-details')
+const claimExpenseHelper = require('../../../../views/helpers/claim-expense-helper')
+
+module.exports = function (router) {
+  router.get('/apply/repeat/eligibility/:referenceId/new-claim/same-journey-as-last-claim', function (req, res, next) {
+    UrlPathValidator(req.params)
+    var referenceAndEligibilityId = referenceIdHelper.extractReferenceId(req.params.referenceId)
+
+    getLastClaimDetails(referenceAndEligibilityId.reference, referenceAndEligibilityId.id)
+      .then(function (lastClaimDetails) {
+        return res.render('apply/eligibility/new-claim/same-journey-as-last-claim', {
+          referenceId: req.params.referenceId,
+          lastClaimDetails: lastClaimDetails,
+          claimExpenseHelper: claimExpenseHelper
+        })
+      })
+      .catch(function (error) {
+        next(error)
+      })
+  })
+
+  router.post('/apply/repeat/eligibility/:referenceId/new-claim/same-journey-as-last-claim', function (req, res, next) {
+    UrlPathValidator(req.params)
+    var referenceAndEligibilityId = referenceIdHelper.extractReferenceId(req.params.referenceId)
+
+    try {
+      new SameJourneyAsLastClaim(req.body['same-journey-as-last-claim']) // eslint-disable-line no-new
+
```

Comment:
Should this class be instantiated in this way?  I can't see anywhere else where this class is used.  Could achieve the same effect in a function unless there is another use for this class